.github/workflows/build_and_test.yml

```diff
@@ -4,24 +4,24 @@ on: [push, pull_request]
 
 jobs:
   build_and_test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     steps:
     - name: Install Java
       run: |
         sudo apt-get update -qq
         sudo apt-get install -y default-jdk gradle
     - uses: actions/checkout@v2
-    - uses: ros-tooling/setup-ros@0.2.1
+    - uses: ros-tooling/setup-ros@v0.2
       with:
-        required-ros-distributions: dashing
+        required-ros-distributions: galactic
```

Comment:
this will introduce conflicts with changes in https://github.com/osrf/ros2_java/

Reply:
this conflicts don't matter much, I'm ok with them